labman/gui/handlers/process_handlers/quantification_process.py

```diff
@@ -37,9 +39,43 @@ def post(self):
             pc = plate.plate_configuration
             concentrations = QuantificationProcess.parse(
                 file_content, rows=pc.num_rows, cols=pc.num_columns)
+
+            names = np.empty_like(plate.layout, dtype='object')
+            blanks = np.zeros_like(plate.layout, dtype=bool)
+
+            for i, full_row in enumerate(plate.layout):
```

Comment:
For future use, can we add a comment line to this block to clarify its purpose? (which I think here is just to get the names and blank status, right?)